src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorCSharpFormattingBenchmark.cs

```diff
@@ -45,7 +45,7 @@ public class RazorCSharpFormattingBenchmark : RazorLanguageServerBenchmarkBase
     public int Blocks { get; set; }
 
     [ParamsAllValues]
-    public InputType InputType { get; set; }
+    internal InputType InputType { get; set; }
```

Comment:
Can we leave the members as public, and just change types? Makes maintenance in future easier